internal/store/x509_auth_database_users.go

```diff
@@ -65,6 +70,17 @@ func (s *Store) SaveX509Configuration(projectID, certificate string) (*atlas.Cus
 	}
 }
 
+// GetUserCertificates retrieves the current user managed certificates for a database user
+func (s *Store) GetUserCertificates(projectID, username string) ([]atlas.UserCertificate, error) {
```

Comment:
similar nit to above, it can be confusing just saying "user" as we have DB users and Atlas users